src/ukrainian-vareniki.md

```diff
@@ -0,0 +1,34 @@
+# Ukrainian Vareniki
+
+(Vareniki/Dumplings) Vareniki are popular all over the world, but aboard they have different names. 
+
+- ⏲️ Prep: 30 mins
+- 🍳 Cook time: 25 mins  
+- 🍽️ Servings: 45
+
+## Ingredients 
+
+- 18 ounces potatoes
+- salt and freshly ground black pepper
```

Comment:
```suggestion
```

No need for salt and pepper in the ingredients list. Mentioning in the directions is good enough.